### PR TITLE
Add fake cap gun to mime & clown uplink

### PR DESCRIPTION
**Pull request by Slava0135**
## About the PR

Adds fake cap gun, it can shoot both cap and magnum bullets. Available in mime and clown uplink.
Idea by @Nairodian 

**Media**

Peek 2023-02-08 15-03

**Changelog**

:cl:
- add: added fake cap gun, can be purchased in clown and mime uplink

### CODE REVIEW

**Comment by deltanedas** [comment]
10tc is way too expensive for a "look its a cap gun, ignore that the guy i killed has bullet wounds and ihnore the magnum cartridges!"

**Comment by GoodWheatley** [comment]
10 TC is way too expensive, especially considering that this is exclusive to mimes and clowns (who will have this blow up in their face) you'll need to buy your own lethal bullets for this. 6 TC would be a better price.

**Comment by Slava0135** [comment]
> 10 TC is way too expensive, especially considering that this is exclusive to mimes and clowns (who will have this blow up in their face) you'll need to buy your own lethal bullets for this. 6 TC would be a better price.

it can't be cheaper than normal revolver (but it doesn't have bullets in it so maybe you can lower it to 8)

**Comment by brainfood1183** [comment]
I think in order for this to work it owuld have to be able to fire both caps and bullets

**Comment by GoodWheatley** [comment]
> I think in order for this to work it would have to be able to fire both caps and bullets
> 
> otherwise it isnt really that beneficial.

That's exactly what it does

**Comment by PixelTheKermit** [comment]
"oh you have a cap gun? Hm mind if I test it with real bullets so I'm sure it's a real cap gun?"
I hate how people will probably start doing this. Anyways cool PR

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -577,6 +577,20 @@
...
+  id: RevolverCapGunFake
+  name: cap gun
+  description: Looks almost like the real thing! Ages 8 and up.
+  components:
+    - type: RevolverAmmoProvider
+      whitelist:
+        tags:
+          - CartridgeCap
+          - SpeedLoaderCap
+          - CartridgeMagnum
```
Comment (GoodWheatley) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:591]: This should probably start with magnum cartridges already loaded so you don't have to also buy a speedloader to make it usable

**Pull request merged**